cmd/dt/pull.go

```diff
@@ -103,5 +107,7 @@ func newPullCommand() *cobra.Command {
 		},
 	}
 	cmd.PersistentFlags().StringVar(&outputFile, "output-file", outputFile, "generate a tar.gz with the output of the pull operation")
+	cmd.PersistentFlags().StringVar(&imagesDir, "images-dir", imagesDir,
+		"directory where the images will be pulled to. If not empty, it overrides the default images directory inside the chart directory")
```

Comment:
Is this description accurate? I thought the images folder was now outside the chart directory after the refactoring recently done.

Reply:
This only applies to the pull/push commands, where there is no concept of wrap bundle, so the images are still placed in the chart dir. The refactoring was for the wrap/unwrap, where we control the format of the bundle.

Reply:
Ah, right I recall it now. I guess it makes sense. I wonder if the images-dir should be mandatory. Anyways, users will tell. Looks good then.